canonical-web-and-design/ubuntu.com
# WIP add /pricing/devices page

## Done

- Added a new page at /pricing/devices, based on this [copy doc](https://docs.google.com/document/d/1uiKNaT59tk2rvjuM24vpqVblcNO90PmE4wUlq9A9Rsk/edit#heading=h.l83dv5879ajz)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Visit: http://0.0.0.0:8001/pricing/devices
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [copy doc](https://docs.google.com/document/d/1uiKNaT59tk2rvjuM24vpqVblcNO90PmE4wUlq9A9Rsk/edit#heading=h.l83dv5879ajz), and is laid out correctly.

## Issue / Card

Fixes #5865
